### DISCUSSION TITLE
How can I use `scrollTo` to scroll `<OverlayScrollbarsComponent/>` using javascript?

### DISCUSSION THREAD

**Discussion by lumenwrites** [KingSora/OverlayScrollbars#496] [Q&A] [answered]
Hi! Before switching to OverlayScrollbars, my components were made scrollable with `overflow-y:scroll;`. When I wanted to scroll my component using javascript, I would create a ref to the component's div, and then do:
```
myComponentRef.current?.scrollTo(0, 0)
```
But this doesn't work with `OverlayScrollbars`.

If I do this:
```
myOverlayScrollbarsComponentRef.current?.scrollTo(0, 0)
```

I get an error telling me that `scrollTo` is not a function.

I also tried:
```
myOverlayScrollbarsComponentRef.current?.getElement().scrollTo(0, 0)
```

And that didn't work either (there's no error, but the scrolling doesn't work either).

Can you please help me to figure out how to do this correctly?

**Comment by KingSora** [comment, marked as answer]
@lumenwrites good day :)

Please try out:
```
overlayScrollbarsComponentRef.current?.osInstance().elements().viewport.scrollTo(0, 0);
```

**Reply by lumenwrites** [reply]
This worked! Thank you so much!
